From 6e3d91964a64dbacfddc4c7a38c64fd221fcdd2b Mon Sep 17 00:00:00 2001
From: Ariel D'Alessandro <ariel.dalessandro@collabora.com>
Date: Wed, 16 Jul 2025 18:44:44 -0300
Subject: [PATCH] update_node_binaries: Update nodejs version to v22.15.0

Update the NodeJS version to match exactly the one shipped in these
images.

Upstream-Status: Inappropriate [specific to our build setup]
Signed-off-by: Ariel D'Alessandro <ariel.dalessandro@collabora.com>
---
 third_party/node/update_node_binaries | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/node/update_node_binaries b/third_party/node/update_node_binaries
index 4aab805844..5c3714f574 100755
--- a/third_party/node/update_node_binaries
+++ b/third_party/node/update_node_binaries
@@ -19,7 +19,7 @@ set -eu
 cd "$(dirname "$0")"

 BASE_URL="https://nodejs.org/dist"
-NODE_VERSION="v22.11.0"
+NODE_VERSION="v22.15.0"

 upload=false  # Default value

